scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
+
+from CoolProp import HumidAirProp as hap
+import pandas as pd
+
+import props
+
+
+results = {'Pressure (Pa)': [], 'Temperature (K)': [],
+					'Ralative Humidity': [], 'Schmidt Mills': [],
+					'Schmidt Marrero': []}
+
+
+# Gets relevant values and calculates Schmidt number
+def get_schmidt(p, t, rh):
+	mills = props.get_d_12(p, t, 'Mills')
+	marrero = props.get_d_12(p, t, 'Marrero')
+
+	# Sometimes errors arise with coolprop from high relative humidities
+	try:
+		rho = 1/hap.HAPropsSI('Vha', 'P', p, 'T', t, 'RH', rh)
```

Comment:
There is a function called `props.get_rho_m(p, t, t_dp)` that you should use here. Just convert `rh` to `t_dp` using the new function you are going to write; i.e. `props.get_tdp(p, t, rh)`.